### PR TITLE
Fix load addon translations: last addon translations wins

### PR DESCRIPTION
**Pull request by giuliaghisini**

### CODE REVIEW

**Review by erral — APPROVED** [review]
Documentation-wise, this is what I was saying.

File: docs/source/addons/i18n.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   html_meta:
-    "description": "Internationalize your add-on and override translations"
-    "property=og:description": "Internationalize your add-on and override translations"
-    "property=og:title": "Add-on Internationalization"
-    "keywords": "Internationalization, i18n, add-on"
+    'description': 'Internationalize your add-on and override translations'
+    'property=og:description': 'Internationalize your add-on and override translations'
+    'property=og:title': 'Add-on Internationalization'
+    'keywords': 'Internationalization, i18n, add-on'
 ---
```
Comment (stevepiercy) [inline docs/source/addons/i18n.md:8]: Please revert the above changes, and adjust your editor's settings not to reformat quotes.

File: docs/source/addons/i18n.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Your add-on has a `locales` folder with a `.pot` file.
 
 In the context of your project, run `yarn i18n` to merge the add-on translations with the ones of your project.
 
-
```
Comment (stevepiercy) [inline docs/source/addons/i18n.md:31]: Please do not remove deliberate white space. It makes it easier for authors to quickly find sections. Revert.

File: docs/source/addons/i18n.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Your add-on has a `locales` folder with a `.pot` file.
...
 ## Override translations
 
-The workflow allows you to override translations from your project scope, so the project translations "always win".
+The workflow allows you to override translations from your project scope, so the addon's translations "always win".
```
Comment (stevepiercy) [inline docs/source/addons/i18n.md:33]:
```suggestion
The workflow allows you to override translations from your project scope, so the add-on's translations "always win".
```

File: docs/source/addons/i18n.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ Your add-on has a `locales` folder with a `.pot` file.
...
+If you have multiple addons installed in your project, the translations are loaded in the order of addons definitions, so the last addon wins.
```
Comment (stevepiercy) [inline docs/source/addons/i18n.md:34]:
```suggestion
If you have multiple add-ons installed in your project, the translations are loaded in the order of the add-ons' definitions, so the last defined add-on wins.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor MyST and English syntax and grammar fixes. Otherwise looks good. Thank you!

File: docs/source/addons/i18n.md [hunk 5]
```diff
@@ -31,4 +31,5 @@ In the context of your project, run `yarn i18n` to merge the add-on translations
 
 ## Override translations
 
-The workflow allows you to override translations from your project scope, so the project translations "always win".
+The workflow allows you to override translations from your project scope, so the add-on's translations "always win".
+If you have multiple add-ons installed in your project, the translations are loaded in the order of add-ons' definitions, so the last defined add-on wins.
```
Comment (stevepiercy) [inline docs/source/addons/i18n.md:35]: Sorry, I missed this on my first pass. I approve once it is fixed. Thank you!

```suggestion
If you have multiple add-ons installed in your project, the translations are loaded in the order of your add-ons' definitions, so the last defined add-on wins.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
One more correction, that I missed on my first pass.

**Review by stevepiercy — APPROVED** [review]
Thank you!

**Review by sneridagh — APPROVED** [review]
LGTM, but I'd like @davisagli and @danalvrz to take a look first, since they were the last that touched that the last time (I think for a similar problem).

File: docs/source/addons/i18n.md [hunk 6]
```diff
@@ -31,4 +31,5 @@ In the context of your project, run `yarn i18n` to merge the add-on translations
 
 ## Override translations
 
-The workflow allows you to override translations from your project scope, so the project translations "always win".
+The workflow allows you to override translations from your project scope, so the add-on's translations "always win".
+If you have multiple add-ons installed in your project, the translations are loaded in the order of your add-ons' definitions, so the last defined add-on wins.
```
Comment (davisagli) [inline docs/source/addons/i18n.md:35]:
```suggestion
If you have multiple add-ons installed in your project, the translations are loaded in the order your add-ons are listed in `package.json`. If two add-ons provide different translations for the same message, then the last defined add-on wins.

When running `yarn i18n` in the context of your project, the project's own locales are processed last and can override translations from any add-on.
```

**Review by davisagli — CHANGES_REQUESTED** [review]
The code fix looks right, but I'm proposing a fix to the docs.

File: docs/source/addons/i18n.md [hunk 7]
```diff
@@ -31,4 +31,6 @@ In the context of your project, run `yarn i18n` to merge the add-on translations
 
 ## Override translations
 
-The workflow allows you to override translations from your project scope, so the project translations "always win".
+If you have multiple add-ons installed in your project, the translations are loaded in the order your add-ons are listed in `package.json`. If two add-ons provide different translations for the same message, then the last defined add-on wins.
```
Comment (stevepiercy) [inline docs/source/addons/i18n.md:34]: One more nitpick. One sentence per line, and this time I *promise* it is good to go. I apologize for my oversight.

```suggestion
If you have multiple add-ons installed in your project, the translations are loaded in the order your add-ons are listed in `package.json`.
If two add-ons provide different translations for the same message, then the last defined add-on wins.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
Thank you for your patience with me!

**Review by davisagli — APPROVED** [review]

**Pull request merged**